### DISCUSSION TITLE
ExpiredAuthSessionError: Both token and refresh token have expired. Your request was aborted.

### DISCUSSION THREAD

**Discussion by wiwat-p** [nuxt-community/auth-module#1514] [Support Requests]
Hi, I have a question.

I try to login with some account.

Some account have got short token it fine.

But some account got long token we have a problem.

After login as account long token `this.$auth.loggedIn` is `false`

This is my code.

     this.$auth.loginWith('local', {
          data: params
     }).then(async (response: any) => {
     try {   
            const verify = await this.$axios.$post(URL, { data: null }) }  // When we call axios we got error
     } catch (error) {
            console.log('error -> ', error) // <- We got error here
     }

**Comment by devzom** [comment]
@wiwat-p

What's Your nuxt.config.js $auth config?
